### ISSUE TITLE
Dante Virtual Soundcard is Apple Silicon Native

### ISSUE THREAD

**Issue by neilph** [ThatGuySam/doesitarm#918]
Replace each second line your own text. 

**The official name of the app**
Dante Virtual Soundcard

**Is there a supported version available on a stable release channel?**
Yes

**Proposed New App Status**
Yes, Native Apple Silicon Support

**Related Issue Tracker Links or discussion**

**An Official App Download Page**
https://my.audinate.com/support/downloads/download-latest-dante-software

**Additional context**
Add any other context or screenshots about the app request here. 

**Full Screenshot with the App and Activity Monitor(if self reporting) or [Silicon Info](https://github.com/billycastelli/Silicon-Info)**
![MicrosoftTeams-image (15)](https://user-images.githubusercontent.com/102574488/160543529-6bde417d-11dc-47ae-b6c0-2a44725279b3.png)

**Comment by ThatGuySam** [comment]
@neilph Thanks for the update! 

Feel free to make a pull request using the [App Addition Template](https://github.com/ThatGuySam/doesitarm/blob/master/.github/PULL_REQUEST_TEMPLATE/app_addition_template.md) otherwise I'll add it when time permits.

**Comment by arnvid** [comment]
There is a bug in the installer which makes it "require" Rosetta to be installed.. A simple edit of the pkg file allows for native M1 install w/o Rosetta.

**Comment by ThatGuySam** [comment]
@arnvid What's the edit?

**Comment by ThatGuySam** [comment]
@neilph Update is now live on the site! 

https://doesitarm.com/app/dante-virtual-soundcard

Feel free to comment on this issue for any future updates. 

Until then, you're welcome to browse through the other [M1 Issues](https://github.com/ThatGuySam/doesitarm/issues?q=is%3Aopen+is%3Aissue+label%3A%22Needs+M1+Testing%22) to help with missing info on other apps.

**Comment by arnvid** [comment]
> @arnvid What's the edit?
@ThatGuySam - the fix that was done and communicated to Audinate for v4.2.3.1 was as following:
1. Unpack the pkg - pkgutil --expand DanteVirtualSoundcard.pkg dvs-expand/

2. Patch the Distribution file

`
diff -bpruN dvs-expand/Distribution dvs-expand-fix/Distribution
--- dvs-expand/Distribution    2022-03-16 13:35:33.000000000 +0100
+++ dvs-expand-fix/Distribution    2022-09-08 14:16:34.000000000 +0200
@@ -1,5 +1,6 @@
 <?xml version=“1.0” encoding=“UTF-8"?>
 <installer-gui-script authoringTool=“Packages” authoringToolVersion=“1.2.9" authoringToolBuild=“603” minSpecVersion=“1.0">
+    <options customize=“never” require-scripts=“false” hostArchitectures=“arm64”/>
     <options rootVolumeOnly=“true” customize=“never”/>
     <volume-check script=“volume_check()“/>
     <!--+==========================+
`
3. Repack the pkg pkgutil --flatten dvs-expand-fix/ DanteVirtualSoundcard-fixed.pkg
4. Install

This was done after Audinate Customer Support explained that they had missed a switch when building the installer.

**Comment by ThatGuySam** [comment]
@arnvid Gracias! 🙏

People from all over reference the site and this repo so I just wanted to leave some breadcrumbs for anyone using Dante on Apple SiIlcion, from an Audio Engineer who's been stuck on this problem for 8 hours to a Touring Engineer who needs this fix because they're going live to an audience of 10k in 30 minutes. 

For those not familiar, the mentioned commands are done in Terminal on macOS. 
```
pkg - pkgutil --expand DanteVirtualSoundcard.pkg dvs-expand/
```

```
diff -bpruN dvs-expand/Distribution dvs-expand-fix/Distribution
```

```
pkg pkgutil --flatten dvs-expand-fix/ DanteVirtualSoundcard-fixed.pkg
```
